lib/lambda-worker/lambda-worker.ts

```diff
@@ -16,6 +16,17 @@ const MINIMUM_MEMORY_SIZE = 1024;
 const MINIMUM_LAMBDA_TIMEOUT = cdk.Duration.seconds(30);
 
 export class LambdaWorker extends cdk.Construct {
+  // The ARN of the queue messages for this LambdaWorker to process
+  // should be placed on.
+  public lambdaQueueArn: string;
+
+  // The URL of the queue messages for this LambdaWorker to process
+  // should be placed on. If you have passed in an option subscrip topic,
+  // this queue will have automatically been subscribed to that topic.
+  // If you have not supplied a topic, then you will need to route
+  // messages to the queue at lambdaQueueUrl.
+  public lambdaQueueUrl: string;
+
```

Comment:
Expose the `lambdaQueueArn` and `lambdaQueueUrl`  so that we can chain workers, with one writing messages to the queue of the next.

I did consider exposing the complete SQS construct which would allow the user to read both of these properties. However, that would give the client full access to modifying the queue, perhaps in ways we don't want.